examples/README.md

```diff
@@ -4,15 +4,15 @@ This directory contains javascript and typescript examples for node.js, browser,
 
 - Node.js examples
   - Prerequisites
-    - [node](https://nodejs.org/en/) installed
+    - [node](https://nodejs.org/en/) installed, at least version 16 is recommended
     - Run `npm install` in this directory
     - Change variables in [./env.mjs](env.mjs) to configure connection to your InfluxDB instance. The file can be used as-is against a new [docker InfluxDB v2.3 OSS GA installation](https://docs.influxdata.com/influxdb/v2.3/get-started/)
   - Examples are executable. If it does not work for you, run `npm run esr EXAMPLE.ts`.
   - [write.mjs](./write.mjs)
     Write data points to InfluxDB.
   - [query.ts](./query.ts)
     Query InfluxDB with [Flux](https://docs.influxdata.com/influxdb/latest/get-started/).
-  - [queryWithParams.ts](./queryWithParams.ts)
+  - [queryWithParams.mjs](./queryWithParams.mjs)
```

Comment:
Please also change the `[scripts/server.js](./scripts/server.js)` to `[scripts/server.mjs](./scripts/server.mjs)`.

Reply:
nice catch :+1: